File: user-web/src/main/webapp/js/service/myOrderService.js
//服务层
app.service('myOrderService', function ($http) {
    //查找我的订单
    this.findAllByUserId = function () {
        return $http.get('../order/findAllByUserId.do');
    };

    //确认收货
    this.confirmOrder = function (orderId) {
        return $http.get('../order/confirmOrder.do?orderId='+orderId);
    };
});

File: order-interface/src/main/java/com/jx/order/service/MyOrderService.java
package com.jx.order.service;


import com.jx.grouppojo.MyOrder;
import com.jx.pojo.TbPayLog;

import java.math.BigInteger;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface MyOrderService {
    /**
     * 根据用户名查找对应的所有订单
     * @param userId 用户名
     * @return
     */
	MyOrder findAllByUserId(String userId);

    /**
     * 确认收货
     * @param orderId  订单的Id
     */
	void confirmOrder(Long orderId);

    /**
     * 根据订单Id去查找未付款的订单
     * @param orderId 订单Id
     * @return
     */
	TbPayLog findByOrderId(String orderId);
}
